data/leapfrog-technology.json

```diff
@@ -3,6 +3,8 @@
   "establishedAt": 1262283300,
   "location": "Charkhal Rd, Dillibazar, Kathmandu",
   "website": "https://www.lftechnology.com/",
+  "email": "hello@lftechnology.com",
+  "phone": null,
```

Comment:
NOTE: Couldn't find any phone number at all 🤨

Reply:
01-4519766
Check this